app/controllers/users/applications_controller.rb

```diff
@@ -26,9 +29,11 @@ def new
     end
 
     def edit
+      authorize application
```

Comment:
I'm curious why these aren't before_actions. (There may be a totally legit reasons, of course, but I understood that as the Ruby Way)